### DISCUSSION TITLE
0.3.4

### DISCUSSION THREAD

**Discussion by schickling** [contentlayerdev/contentlayer#497] [Announcements]
ℹ️ **[TLDR]** Many smaller bug fixes, improvements and updated dependencies

## Improvements

- Added `aspectRatio` property to `ImageFieldData` for `type: image` fields
- Added [images example](https://github.com/contentlayerdev/contentlayer/tree/main/examples/next-images) 
- Support for esbuild 0.18 (closes #496)
- Upgraded various dependencies (incl. `yaml` package - closes  #488)
- Added MIT license to all sub packages in the Contentlayer mono repo (closes #482)
- next-contentlayer should have contentlayer as peerDependency #447

## Bug fixes

- Fix Bug in calculation of "_raw.flattenedPath" (closes #487)

<hr /><em>This discussion was created from the release <a href='https://github.com/contentlayerdev/contentlayer/releases/tag/'>0.3.4</a>.</em>

**Comment by Svish** [comment]
In the images example, it uses `type: 'image'`, but is that documented anywhere? I'm guessing it should be listed on the [Field Types Reference](https://www.contentlayer.dev/docs/reference/source-files/field-types-defe41e9), but can't find it anywhere, and can't find anything about it when searching either. 🤔

**Comment by dstroot** [comment]
@schickling Hi there! This library is a godsend for MDX blogs with Nextjs. Phenomenal.  Thank you so much. There were regular monthly releases until June 2024 and nothing since.  Is this library still supported?  Could you provide a little guidance? I switched my personal blog over, but I am now getting concerned.  Recent remark and rehype plugins are breaking content layer.

**Reply by schickling** [reply]
Hey there! Thanks for your kind message and question. I've recently provided an tentative update [here](https://github.com/contentlayerdev/contentlayer/issues/429#issuecomment-1731298319):

> As you might have seen, [Stackbit recently got acquired by Netlify](https://www.netlify.com/blog/netlify-acquires-stackbit-to-bring-no-code-creation-to-its-platform/) which (at least for now) means that the sponsorship of Contentlayer has been paused. I'm hoping to hear back from Netlify soon on the continued sponsoring & development of Contentlayer as there is a lot more work to be done! 💪

**Reply by dstroot** [reply]
Thanks! Got it!
